account-kit/react/src/components/dialog/dialog.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback, useEffect, useState, type ReactNode } from "react";
+import { createPortal } from "react-dom";
+import { RemoveScroll } from "react-remove-scroll";
+import { FocusTrap } from "./focustrap.js";
+
+type DialogProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  children: ReactNode;
+};
+
+// eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
good news! you shouldn't need to add these anymore if they're not exported from the `index.ts` file